intellij/src/saros/intellij/ui/views/buttons/FollowButton.java

```diff
@@ -106,7 +107,10 @@ public FollowButton() {
   }
 
   private void createMenu() {
-    popupMenu = new JPopupMenu();
+    popupMenu = new JBPopupMenu();
+
+    popupMenu.setForeground(FOREGROUND_COLOR);
```

Comment:
Is this still necessary for a JBPopupMenu ?

Reply:
Yes, as otherwise it still retains the initial color when switching the IDE theme. This is visible between the entries.

As to why all of this is necessary, I have no idea. Same as in #535, this is more like a hackie workaround than a real fix.